Amazon.IonObjectMapper/IonSerializationOptions.cs

```diff
@@ -17,48 +17,116 @@ namespace Amazon.IonObjectMapper
     using System.Collections.Generic;
     using static Amazon.IonObjectMapper.IonSerializationFormat;
 
+    /// <summary>
+    /// Serialization options.
+    /// </summary>
     public class IonSerializationOptions
     {
+        /// <summary>
+        /// Gets option to specify the .NET property naming convention.
+        /// </summary>
         public IIonPropertyNamingConvention NamingConvention { get; init; } = new CamelCaseNamingConvention();
 
+        /// <summary>
+        /// Gets option to specify the serialization format.
+        /// </summary>
         public IonSerializationFormat Format { get; init; } = TEXT;
 
+        /// <summary>
+        /// Gets option to specify the maximum deserialization depth.
+        /// </summary>
         public int MaxDepth { get; init; } = 64;
 
+        /// <summary>
+        /// Gets a value indicating whether to annotate Guids.
```

Comment:
Why is this a value instead of option?

Reply:
StyleCop complains if we say "option" instead of "value".

![image](https://user-images.githubusercontent.com/62349012/125521255-949c4114-8be2-4ea1-bfa1-971dc239fa59.png)